src/components/RessourceMenu/RessourceMenu.tsx

```diff
@@ -49,29 +53,35 @@ export const RessourceMenu = () => {
             <td>
               <b>{Math.floor(ressources.food)}</b> ({valueSmoothener(buildingOutputs.food)})
             </td>
-            <td></td>
-            <td>{Math.floor(ressources.villager)}</td>
-          </tr>
-          <tr>
             <td>
-              <b>Remaining</b>
+              <b>{Math.floor(ressources.villager)}</b>
             </td>
-            <td>{remainingRessources(valueSmoothener(buildingOutputs.gold), Math.floor(ressources.gold))} </td>
-            <td>{remainingRessources(valueSmoothener(buildingOutputs.wood), Math.floor(ressources.wood))}</td>
-            <td>{remainingRessources(valueSmoothener(buildingOutputs.stone), Math.floor(ressources.stone))}</td>
-            <td>{remainingRessources(valueSmoothener(buildingOutputs.food), Math.floor(ressources.food))}</td>
-            <td></td>
-            <td></td>
           </tr>
+          <>
+            {selected && (
+              <tr className="ressource-menu-extended-items">
+                <td>
+                  <b>⏳</b>
+                </td>
+                <td>{remainingRessources(valueSmoothener(buildingOutputs.gold), Math.floor(ressources.gold))} </td>
+                <td>{remainingRessources(valueSmoothener(buildingOutputs.wood), Math.floor(ressources.wood))}</td>
+                <td>{remainingRessources(valueSmoothener(buildingOutputs.stone), Math.floor(ressources.stone))}</td>
+                <td>{remainingRessources(valueSmoothener(buildingOutputs.food), Math.floor(ressources.food))}</td>
+                <td></td>
+              </tr>
+            )}
```

Comment:
why are they only displayed when a tile is selected? imo it would be useful to always see these stats

Reply:
My intention was to maximize screen size when doing noting or especially when exploring the map. So when you explore further areas, you know what resources you have and with what ratio they "move", but the remaining time is only needed when you are about to build new buildings and stuff. That was my thought, the initial layout was only reduced in landscape, but I expanded it to portrait too. 
But I could understand if the time is a more important parameter for you, so I can easily revert the changes.